src/System.Private.ServiceModel/tools/setupfiles/sync-pr.cmd

```diff
@@ -0,0 +1,43 @@
+@echo off 
+setlocal
+
+REM 
+REM This calls a web service, specified on the command line, which syncs a 
+REM remote server's repo to the correct PR. 
+REM This script is meant to be called from a GitHub Pull Request. 
+REM If you must, you can fake it by setting the 'ghprbPullId' environment variable
+REM 
+
+set __args=%*
+
+echo.
+echo  WCF Remote Github Repo sync script
+
+if '%__args%'=='' (
+    echo   [%~n0] A URL must be specified for the pull request synchronization URL
+    echo     Usage:  %~n0 [sync url]
+    echo   sync-url - URL on remote server for PR synchronization
+    echo.
+    echo   Example:  %~n0 http://wcfci-sync-server/repo/sync.ashx
+    set __EXITCODE=1
+    goto done
+)
+
+set __SYNC_HOST_URL=%1
+
+if '%ghprbPullId%'=='' (
+    echo   [%~n0] This script should only be called only from the context of a GitHub Pull Request
```

Comment:
Can we pass this in as a parameter (with optional substitution of environment variable)?  Environment variables as required input parameters are often problematic.  If there is some reason this is not possible, then show_usage should explain what environment variables are required.